src/traffic_layout/engine.cc

```diff
@@ -362,7 +363,7 @@ LayoutEngine::remove_runroot()
     return;
   }
 
-  std::string clean_root = path;
+  std::string clean_root = std::move(path);
```

Comment:
Not sure what the intent here was.  Does this code want a copy of path? If so, your std::move is counter to that goal.  If not, then this is an unnecessary temp.

Reply:
I think the intent was to describe the data contained in the string, even though it's the same string object being modified. I think `clean_root` should be a reference - there's no need to copy or move.